docs/spec/parameters.md

```diff
@@ -5,21 +5,27 @@ Each version of the protocol will define a set of protocol rules and parameters
 | Protocol Parameter          | Description                                                                   | Suggested Defaults |
 |-----------------------------|-------------------------------------------------------------------------------|:-------------------|
 | `HASH_ALGORITHM` { #hash-algorithm }       | Algorithm for generating hashes of protocol-related values.    |             SHA256 |
-| `HASH_PROTOCOL` { #hash-protocol }       | Protocol for generating hash representations in Sidetree implementations, using the [`HASH_ALGORITHM`](#hash-algorithm) | [Multihash](#multihash) |
+| `HASH_PROTOCOL` { #hash-protocol }       | Protocol for generating hash representations in Sidetree implementations, using the [`HASH_ALGORITHM`](#hash-algorithm) | [Multihash](https://multiformats.io/multihash/) |
 | `DATA_ENCODING_SCHEME` { #data-encoding-scheme } | Encoding selected for various data (JSON, hashes, etc.) used within an implementation, the output of which ****MUST**** be in ASCII format. | Base64URL |
 | `JSON_CANONICALIZATION_SCHEME` { #json-canonicalization-scheme } | The scheme selected for canonicalizing JSON structures used throughout the specification. | [JCS](https://tools.ietf.org/html/draft-rundgren-json-canonicalization-scheme-17) |
-| `KEY_ALGORITHM` { #key-algorithm }         | Asymmetric public key algorithm for signing DID operations. Must be a valid JWK `crv`.    |          secp256k1 |
-| `SIGNATURE_ALGORITHM` { #sig-algorithm }   | Asymmetric public key signature algorithm. Must be a valid JWS `alg`.                    |             ES256K |
-| `CAS_PROTOCOL` { #cas-protocol }       | The CAS network protocol used within an implementation.            |               IPFS |
+| `KEY_ALGORITHM` { #key-algorithm }         | Asymmetric public key algorithm for signing DID operations. Must be a valid JWK `crv`. | secp256k1 |
+| `SIGNATURE_ALGORITHM` { #sig-algorithm }   | Asymmetric public key signature algorithm. Must be a valid JWS `alg`. |              ES256K |
+| `CAS_PROTOCOL` { #cas-protocol }       | The CAS network protocol used within an implementation. | [IPFS](https://github.com/ipfs/specs) |
 | `CAS_URI_ALGORITHM` { #cas-uri-algorithm }       | Algorithm for generating unique content-bound identifiers for the implementation-selected CAS protocol                      |           IPFS CID |
-| `COMPRESSION_ALGORITHM` { #compression-algorithm } | File compression algorithm                             |                ZIP |
-| `REVEAL_VALUE` { #reveal-value } | Cryptographically random value to be revealed in the next operation. |  100 bytes |
+| `COMPRESSION_ALGORITHM` { #compression-algorithm } | File compression algorithm                             |                        ZIP |
+| `REVEAL_VALUE` { #reveal-value } | Cryptographic hash of the commitment value. |                                        SHA256 Multihash |
```

Comment:
This parameter is not used in reference application. What is the rationale for stating it as parameter?

Reply:
I added the use of it in the Common Functions section, and need to use it in other places, which I will do another PR for